extension/imos--1.0.sql

```diff
@@ -399,3 +399,15 @@ as $$
     end;
 $$;
 
+-- Run pg_stat_activity as superuser for non-superusers so they can see queries being executed by other users
+create function database_activity() returns setof pg_catalog.pg_stat_activity
```

Comment:
Just poking around and found this even more concise example which doesn't require the loop or variable declaration:

```sql
https://dba.stackexchange.com/questions/58271/how-to-view-the-query-of-another-session-in-pg-stat-activity-without-being-super
```

Probably much of a muchness for a query this tiny though! Happy to merge either way, since I think we've agreed the usual privacy reason for not doing this are moot anyway.

Reply:
Sure - thats a better way of doing it.  I'll make sure it works in vagrant for a new db

Reply:
Yep works just as well. Should be OK to merge